sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -120,6 +120,12 @@ public partial class MatchConditions
         public Azure.ETag? IfMatch { get { throw null; } set { } }
         public Azure.ETag? IfNoneMatch { get { throw null; } set { } }
     }
+    public static partial class ModelSerializer
+    {
+        public static T Deserialize<T>(System.IO.Stream stream, Azure.SerializableOptions? options = null) where T : Azure.IJsonSerializable, new() { throw null; }
+        public static T Deserialize<T>(string json, Azure.SerializableOptions? options = null) where T : Azure.IJsonSerializable, new() { throw null; }
+        public static System.IO.Stream Serialize<T>(T model, Azure.SerializableOptions? options = null) where T : Azure.IJsonSerializable, new() { throw null; }
```

Comment:
Would SerializableOptions be a superset of what's available in the STJ JsonSerializer?

Reply:
Would it change to ModelSerializerOptions?  That could be the answer for cleanly partitioning model serialization options from more general content serialization options.

Reply:
Not exactly because we don't want people to be able to pass in their own property mapping settings since that defeats the purpose of not needing to know how to convert a model to azure wire format.

Reply:
Yeah, conceptually, it feels like in a way ModelSerializerOptions is a subtype of SerializerOptions where a lot of the options have been fixed by the model.  So maybe it's not a subtype so much as an "instance of", where a few options are left un-fixed for the user to specify if needed.  

I do think renaming it to ModelSerializerOptions to map it directly to ModelSerializer would be a nice way to communicate this to users.  That leaves space for a SerializerOptions with more of the options available for callers to designate in more general non-model cases.